OneginiExampleAppSwift/ViewControllers/Login/LoginViewController.swift

```diff
@@ -46,8 +45,8 @@ class LoginViewController: UIViewController {
     }
 
     weak var loginDelegate: LoginViewDelegate?
-    var selectedProfile = ONGUserProfile()
-
+    var selectedProfile: UserProfile! //TODO: that will fail, solve
```

Comment:
todo?

Reply:
Can be removed. Force unwrapping can stay - first selection is done in viewDidLoad.

Reply:
Removed.